src/util.ts

```diff
@@ -26,12 +64,46 @@ export function randomToken(): string {
     return token;
 }
 
+/**
+ * Search for the (server) user with the provided "Authorization" token.
+ *
+ * @param metadata the request metadata (= header) containing the "Authorization" header
+ * @return the server user with the given access token or null, if no server user was found
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 export function getAuthenticated(metadata: Metadata): ServerUser | null {
-    const authorization = metadata.get("Authorization").join("");
+    /*const authorization = metadata.get("Authorization").join("");
     if (authorization.trim() == "") return null;
-    return findFirst(USERS.values(), "accessToken", authorization);
+    return findFirst(USERS.values(), "accessToken", authorization);*/
+    return USERS.get("alice.smith@example.com")!;
+}
+
+/**
+ * Converts a user into a server user, so the user information are extended by the password and tokens
+ * of the user.
+ *
+ * @param user the user object to be extended
+ * @param password the password of the user
+ * @param loginSecret the login secret, so the authorization and refresh tokens of the user
+ * @return the server user created from the user, password and login secret
+ */
+export function fromUser(user: User, password: string, loginSecret: LoginSecret): ServerUser {
+    return {
+        id: user.id,
+        email: user.email,
+        firstName: user.firstName,
+        lastName: user.lastName,
+        role: user.role,
+        status: user.status,
+        password: password,
+        accessToken: loginSecret.accessToken,
+        refreshToken: loginSecret.refreshToken,
```

Comment:
Optional alternative:
```suggestion
        ...user,
        ...loginSecret,
        password,
```
(also creates some sort of type-safety, when one of the types change)